scripts/base/init-bare.zeek

```diff
@@ -2856,6 +2856,8 @@ export {
 		version     : NTLM::Version &optional;
 		## Attribute-value pairs specified by the server
 		target_info : NTLM::AVs &optional;
+
+		challenge   : string;
```

Comment:
IMO this should be placed after the `flags` field and the `Assign()` offsets for the challenge record updated within `ntlm-analyzer.pac`. This would allow the layout to stay the same as it appears on the wire and docs

 Also the documentation string would be useful. This is from the NTLM docs: "A 64-bit value that contains the NTLM challenge.".

I haven't fully understood why you prefer to have it as (byte)string rather than a count. I believe you stated that somewhere, but I'm failing to find the reference. Mind providing that as input?

Reply:
Will do :) 
I used byte(string) instead of count because I had to convert it from big endian (network order) to little endian/

Reply:
Hmm, I'm a bit confused. The challenge should've been parsed as little-endian and converted to `count` - did something unexpected happen?

The glossary for NTLM describes the `nonce` as a number and the `challenge` as a nonce, so might be nice if we keep the `uint64_t` / `count`.

Reply:
When I use `uint64` and `count` in `ntlm-protocol.pac` and `init-bare.zeek` respectively instead of byte(string) the challenge is in reverse endianess:
![image](https://user-images.githubusercontent.com/27895842/202311199-57e17abc-310a-4cab-9803-b9315ac133d4.png)

Reply:
That seems mostly a "display" problem, but if bytestring is easier to work with can leave that.